xrpl/asyncio/wallet/wallet_generation.py

```diff
@@ -50,6 +55,14 @@ async def generate_faucet_wallet(
     .. # noqa: DAR402 exception raised in private method
     """
     faucet_url = get_faucet_url(client.url, faucet_host)
+
+    if faucet_url == _HOOKS_V2_TEST_FAUCET_URL and wallet is not None:
+        raise XRPLFaucetException(
+            """Currently the Hooks Testnet v2 faucet has no way of funding a given wallet.
+            If you need to do that, you can create a new funded account and have it send
+            a payment transaction to the existing account."""
```

Comment:
```suggestion
            "Currently the Hooks Testnet v2 faucet has no way of funding a given wallet. "
            "If you need to do that, you can create a new funded account and have it send "
            "a payment transaction to the existing account."
```
The triple quotes will include the `\n`s in the message.